### DISCUSSION TITLE
Conformal Coating -- what's the best coating, what's the best application method?

### DISCUSSION THREAD

**Discussion by drspangle** [rbaron/b-parasite#82] [Ideas]
@rbaron I was puzzling over the application of UV-resin as mentioned in #79 and was wondering if you might have any insight into the thickness of the coating and the impact on the capacitive sensor. I had been searching around the internet and there's very little I could find on this, as I think most people would opt for potting the entire device in UV resin instead of using it as a conformal. By dipping the sides and brushing on a very thin layer over the solder mask, I'm sort of doing both, encapsulating the board (minus the battery) but limiting the thickness over the actual sensing area at the business end. 

I'm concerned I might not understand the implications of my choice, and/or might be focusing on the wrong things. Any thoughts on this?

**Comment by rbaron** [comment]
@drspangle I've only really used acrylic spray for coating. I filed #79 hoping to hear about other users' experiences.

For acrylic spray coating I've barely noticed a difference in the readings. I would suggest running the [soil_read_loop](https://github.com/rbaron/b-parasite/tree/main/code/nrf-connect/samples/soil_read_loop) firmware sample before/after trying different methods to assess the difference.

**Reply by BenRoe** [reply]
Are there instructions available how to read the  values from the soil_read_loop firmware?
I use a J-Link.

**Reply by rbaron** [reply]
@BenRoe not formal docs, but I wrote about how I do it in https://github.com/rbaron/b-parasite/issues/170
